### PR TITLE
Added logic to load magna header with feature flag

### PR DESCRIPTION
**Pull request by webbyfox**
This changeset adds logic to load the Magna header on any repo which uses directory-components.

To trigger magna header: relevant repo requires two following env vars 

`MAGNA_HEADER=true`
`DIRECTORY_CONSTANTS_URL_GREAT_MAGNA=http://greatcms.trade.great:8020/`

 - [x] Changelog entry added.
 - [x] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [x] (if updating requirements) Requirements have been compiled.
 - [x] (if adding feature flags) Cleaned up old flags
 - [x] (if changing the UI) Add a printscreen to the PR

image

### CODE REVIEW

File: directory_components/static/directory_components/js/dit.components.header.js [hunk 1]
```diff
@@ -119,15 +170,20 @@ dit.components.header = (new function() {
...
-      if (event.key === "ArrowDown" || event.key === "Down") {
-        self.openMenu();
-        self.moveFocusToFirstMenuItem();
-        event.preventDefault();
-      }
+    if (
+      event.key === "Escape" ||
+      event.key === "ArrowUp" ||
+      event.key === "Esc" ||
+      event.key === "Up"
```
Comment (bobmeredith) [inline directory_components/static/directory_components/js/dit.components.header.js:177]: Could use something like if(event.key in {Escape:1, ArrowUp:1, Esc:1, Up:1}) { ...

File: directory_components/static/directory_components/js/dit.components.header.js [hunk 2]
```diff
@@ -119,15 +170,20 @@ dit.components.header = (new function() {
...
+    ) {
+      self.closeMenu();
+      event.preventDefault();
+    }
+    if (event.key === "ArrowDown" || event.key === "Down") {
```
Comment (bobmeredith) [inline directory_components/static/directory_components/js/dit.components.header.js:182]: See above

**Review by bobmeredith — APPROVED** [review]
Looks good.
Is all that scss copied?
There is quite a bit of reformatting. I made a couple of comments, but if it's just a reformat of existing code, probably not worth worrying.

**Comment by webbyfox** [comment]
Yes @bobmeredith, Lewis did mostly lift and shift

**Pull request merged**